NitroxServer/GameLogic/EscapePodManager.cs

```diff
@@ -47,18 +97,33 @@ public EscapePodModel[] GetEscapePods()
 
         private EscapePodModel CreateNewEscapePod()
         {
+            //Had to get the data from somewhere and because gameobjects are inaccessable form outside of the game Had to hardcode the information
+            const string NUTRIENT_BLOCK_DATA = "AggBhgEIABAAGAAiCFVudGFnZ2VkMiQwYTJjZTc2Yi0yNTJlLTRmYzMtOGZhYy01NDA1N2ViOTAxNGY6JGEwNjE1N2NjLThkZTgtNGZlYy04NWE2LTc2YjJhZWUxZTI2M0IkM2Q0NjdiMDEtODExNC00OTExLWJmODgtYWVlMmI3M2JlMGE4SAFQAAIIBBkKFVVuaXR5RW5naW5lLlRyYW5zZm9ybRABOAoPDQAAAAAVAAAAAB0AAAAAEhQNAAAAABUAAAAAHQAAAAAlAACAPxoPDQAAgD8VAACAPx0AAIA/DgoKUGlja3VwYWJsZRABE8I+EAgAEAAYACAAKAEwATgAQAMUChBMYXJnZVdvcmxkRW50aXR5EAECEAALCgdFYXRhYmxlEAEFDQAAAAA=";
+            const string FILTERED_WATER_DATA = "AggBhgEIABAAGAAiCFVudGFnZ2VkMiRkNDdmM2FiZi1kNzMxLTQ4MjktYWY0Yy1kNmNhMzhhODFiZjU6JDIyYjBjZTA4LTYxYzktNDQ0Mi1hODNkLWJhN2ZiOTlmMjZiMEIkM2Q0NjdiMDEtODExNC00OTExLWJmODgtYWVlMmI3M2JlMGE4SAFQAAIIBBkKFVVuaXR5RW5naW5lLlRyYW5zZm9ybRABOAoPDQAAAAAVAAAAAB0AAAAAEhQNAAAAABUAAAAAHQAAAAAlAACAPxoPDQAAgD8VAACAPx0AAIA/DgoKUGlja3VwYWJsZRABE8I+EAgAEAAYACAAKAEwATgAQAMUChBMYXJnZVdvcmxkRW50aXR5EAECEAALCgdFYXRhYmxlEAEFDQAAAAA=";
+            const string FLARE_DATA = "AggBhgEIABAAGAgiCFVudGFnZ2VkMiQ1YzE4ODExNS03NWZhLTQ2NTMtYWYxMi02ZGYzMjFhM2E0NDk6JGY1YjZlYmVkLTlhZDktNGU0Ni1iMGI0LTBmOGQ2NDY3ZTA5MUIkM2Q0NjdiMDEtODExNC00OTExLWJmODgtYWVlMmI3M2JlMGE4SAFQAAIIBBkKFVVuaXR5RW5naW5lLlRyYW5zZm9ybRABOAoPDQAAAAAVAAAAAB0AAAAAEhQNAAAAABUAAAAAHQAAAAAlAACAPxoPDQAAgD8VAACAPx0AAIA/DgoKUGlja3VwYWJsZRABE8I+EAgAEAAYACAAKAEwATgAQAMJCgVGbGFyZRABDg0AAOFEEAAYACUAAAAAFAoQTGFyZ2VXb3JsZEVudGl0eRABAhAA";
```

Comment:
Instead of putting items in the escape pod, why don't we put items into the players inventory when they first join the server.  This would allow everyone to get the starting items.  If you do it that way, I think you might be able to run the routine in client side code (so you don't have crazy stuff like this here)

Reply:
Great idea, didn't think of that :D